### PR TITLE
Resolved #2356, added Somali-language E & T orientation

### LINKED ISSUE #2356: SNAP E & T Letter enhancement: new Somali-language orientation for x127 users
New orientation available for x127 users to schedule Somali-language orientations for ABAWDS.
Assign to me, submitting pull now. 

Also need to remove the following verbiage for all users: "A verification request for proof of contact with E & T within 30 days will also need to be sent to the recipient". It has been clarified from Karen Nelson-Huss, DHS, that a verification request does not need to be sent in this instance.

### PR DESCRIPTION
**Pull request by IlseFerris**
BLIP: "Added Somali-language E & T orientation for Hennepin County users. This orientation begins on 07/12/16, and users can start scheduling clients for this orientation on 07/06/16.
Also removed verbiage for all users: "A verification request for proof of contact with E & T within 30 days will also need to be sent to the recipient". It has been clarified from Karen Nelson-Huss, DHS, that a verification request does not need to be sent in this instance."

### CODE REVIEW

File: Script Files/NOTICES/NOTICES - SNAP E AND T LETTER.vbs [hunk 1]
```diff
@@ -163,20 +163,18 @@ BeginDialog SNAPET_automated_adress_dialog, 0, 0, 306, 275, "SNAP E&T Appointmen
   Text 5, 30, 60, 10, "Appointment Date:"
   Text 180, 75, 50, 10, "Contact phone:"
   Text 5, 50, 105, 10, "Location (select from dropdown):"
-  GroupBox 5, 165, 295, 95, "When is a manual referral needed"
+  GroupBox 5, 165, 295, 70, "When is a manual referral needed"
   Text 15, 180, 275, 20, "If an ABAWD is using banked months, or a student meets criteria under CM0011.18, or receiving E and T services through a Community Based Organization (CBO)."
   Text 10, 10, 50, 10, "Case Number:"
   Text 5, 100, 80, 10, "Manual referral needed:"
   Text 15, 205, 275, 25, "Select a recipient type in the 'Manual referral needed' field, and a manual referral will be created with the information entered into the edit boxes above, and a TIKL will be made for 30 days from the date of manual referral."
-  Text 15, 235, 280, 15, "A verification request for proof of contact with E and T within 30 days will also need to be sent to the recipient."
```
Comment (theVKC) [inline Script Files/NOTICES/NOTICES - SNAP E AND T LETTER.vbs]: Why was this removed @IlseFerris?

File: Script Files/NOTICES/NOTICES - SNAP E AND T LETTER.vbs [hunk 2]
```diff
@@ -386,7 +393,8 @@ DO
 		If AM_PM = "Select one..." then err_msg = err_msg & vbNewLine & "* Select either AM or PM for your appointment time."
 		IF SNAPET_contact = "" then err_msg = err_msg & vbNewLine & "* Enter a contact name."
 		IF SNAPET_phone = "" then err_msg = err_msg & vbNewLine & "* Enter a phone number."
-		If interview_location = "Select one..." then err_msg = err_msg & vbNewLine & "* Enter an interview location."
+		If interview_location = "Select one..." then err_msg = err_msg & vbNewLine & "* Enter an interview location." 
+		If (worker_county_code = "x127" and interview_location = "Somali-language (Sabathani, next Tuesday @ 2:00 p.m.)" AND date < "7/05/2016") then err_msg = err_msg & vbNewLine & "* Somali-language orientation is not available until 07/12/16."
```
Comment (theVKC) [inline Script Files/NOTICES/NOTICES - SNAP E AND T LETTER.vbs]: You're using a plain `date` constant. Is it not possible to send this to folks before their orientation? Should this be tied to a date listed on the dialog?

**Comment by IlseFerris** [comment]
Sorry, I added the reason to the commit and then not here. 
Verification of participation of E & T from the recipient was initially stated to be required for banked months cases, but has since been clarified from Karen Nelson-Huss that the client does not need to provide this verification.

**Comment by theVKC** [comment]
See my notes, @IlseFerris!

**Comment by IlseFerris** [comment]
@theVKC (look at me, tagging you with the right name!)
I have this restriction written in here since x127 users don't select the date of the orientation in the dialog. They select it afterwards. (Lines 405-418).

**Comment by theVKC** [comment]
Sounds good. And the deleted line?

**Comment by IlseFerris** [comment]
I commented on it already, but here you go:
Sorry, I added the reason to the commit and then not here. 
Verification of participation of E & T from the recipient was initially stated to be required for banked months cases, but has since been clarified from Karen Nelson-Huss that the client does not need to provide this verification.

**Comment by theVKC** [comment]
Sounds good, but that's not a GitHub issue, nor is it addressed in your blip. Can you make sure that gets done before I accept this?

**Comment by IlseFerris** [comment]
Done and done!
